segmentation/utils/visualize_test.py

```diff
@@ -10,3 +14,12 @@ def test_visualize_z_scores():
     # Assign random phenotype titles
     pheno_titles = [chr(i) for i in range(ord('a'), ord('z') + 1)]
     plot = visualize.visualize_z_scores(z, pheno_titles)
+
+
+def test_visualize_cells():
```

Comment:
We can add a couple assert statements to make sure the right plot is saved when the different options are provided by the user